orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -297,6 +297,10 @@ def _write_function_body(
       writer.write(f'std::vector<OrtValue> {onnx_op.outputs}')
       writer.writeline(f'({onnx_op.outputs.count});')
 
+      if in_place_param:
+        assert(onnx_op.outputs.count == 1)
+        writer.writeline(f'{onnx_op.outputs}[0] = ort_input_{onnx_op.inputs[0]};')
```

Comment:
could you put a comments why it always inputs[0]? it is ok for inplace op like zero_ now, but we might hit more complicated case in the future when we extend to the _out invoke.